### ISSUE TITLE
Death Knight Starter  Quest 12733 - Char chasing Duellists running downhill  [enhancement]

### ISSUE THREAD

**Issue by marrvin2** [EchoTiger/NewQuestingPack#28]
I just observerd a new  tester DK on quest 12733 where he has to duel  5 Initiates.
The DK choses one of the Initiates  that are running downhill to the field.
My char was chasing him a by foot a long time without a chance to catch up.
So I modified the PursueList to ignore the running NPCs:

```
		<Code File="InteractWith" GossipOptions="1" CollectionDistance="200" NonCompeteDistance="0" PreInteractMountStrategy="Dismount" WaitTime="13000" QuestId="12733" QuestObjectiveIndex="1" >
			<HuntingGrounds>
				<Hotspot X="2345.681" Y="-5686.181" Z="153.9222" />
				<Hotspot X="2393.972" Y="-5740.508" Z="153.9231" />
			</HuntingGrounds>
			<PursuitList>
				<PursueUnit Id="28406" PursueWhen="!UNIT.IsMoving" />
			</PursuitList>
		</Code>

```

**Comment by EchoTiger** [comment]
A temporary blackspot or a collection distance reduction for the duel area should fix it.
I'll add this in once I have another tester go through.

**Comment by marrvin2** [comment]
you are right... the  PursueWhen="!UNIT.IsMoving"    is not a good thing... sometimes the char talks to a duellist and after a few seconds he is looking for a new one ;)
Is it possible to shorten the "time to reach target"  blacklist time from 30 to like 8 seconds ?  that would do the trick I guess.
